hydra-cluster/src/CardanoClient.hs

```diff
@@ -214,21 +135,6 @@ waitForUTxO networkId nodeSocket utxo =
     txOut ->
       error $ "Unexpected TxOut " <> show txOut
 
-waitForTransaction ::
-  NetworkId ->
-  FilePath ->
-  Tx ->
-  IO UTxO
-waitForTransaction networkId socket tx =
-  go
- where
-  ins = Map.keys (UTxO.toMap $ utxoFromTx tx)
-  go = do
-    utxo <- queryUTxOByTxIn networkId socket QueryTip ins
-    if null utxo
-      then go
-      else pure utxo
```

Comment:
Moved to `hydra-node` and renamed as `awaitTransaction`

Cf discussion in https://github.com/input-output-hk/hydra-poc/discussions/453